.github/workflows/release-build-zip-file.yml

```
@@ -16,6 +16,22 @@ on:
         type: boolean
         required: true
         default: false
+  workflow_call:
+    inputs:
+      skip_verify:
+        description: 'Skip verification steps.'
+        type: boolean
+        required: true
+        default: false
```

actionlint (1.7.7)

input "skip_verify" of workflow_call event has the default value "false", but it is also required. if an input is marked as required, its default value will never be used

(events)